app/assets/stylesheets/helpers/_organisations.scss

```diff
@@ -34,124 +34,124 @@
   }
 
   &:visited {
-    color: $text-colour;
+    color: $govuk-text-colour;
   }
 }
 
 .organisation-logo {
   @include organisation-logo-19;
 }
 
-@mixin organisation-logo-border($colour) {
-  border-color: $colour;
+@mixin organisation-logo-border($organisation) {
+  border-color: govuk-organisation-colour($organisation);
 }
 
 .attorney-generals-office {
-  @include organisation-logo-border($attorney-generals-office);
+  @include organisation-logo-border('attorney-generals-office');
```

Comment:
This looks good, but do we need these organisation colours at all anymore? Where are they used? I know some components in the gem use them, but that code isn't reliant on this.

Reply:
I also suspect we can remove this, but I'll need to check a few apps before doing that.